From dc71e2214295ed52fc7d1f155644de2ccec1851d Mon Sep 17 00:00:00 2001
From: Daniel Schramm <38858589+danoslas@users.noreply.github.com>
Date: Sun, 24 Jul 2022 22:12:21 +0200
Subject: [PATCH 12/25] UlozTo.cz duplicate HTTPS in captcha URL (#4176)

* Duplicate 'https' in captcha url fix

* Plugin version number increased
---
 src/pyload/plugins/downloaders/UlozTo.py | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/src/pyload/plugins/downloaders/UlozTo.py b/src/pyload/plugins/downloaders/UlozTo.py
index a6dc41bdc..4527e5554 100644
--- a/src/pyload/plugins/downloaders/UlozTo.py
+++ b/src/pyload/plugins/downloaders/UlozTo.py
@@ -24,7 +24,7 @@ def convert_decimal_prefix(m):
 class UlozTo(SimpleDownloader):
     __name__ = "UlozTo"
     __type__ = "downloader"
-    __version__ = "1.50"
+    __version__ = "1.51"
     __status__ = "testing"
 
     __pattern__ = r"https?://(?:www\.)?(uloz\.to|ulozto\.(cz|sk|net)|bagruj\.cz|zachowajto\.pl|pornfile\.cz)/(?:live/)?(?P<ID>[!\w]+/[^/?]*)"
@@ -146,12 +146,6 @@ class UlozTo(SimpleDownloader):
 
                 xapca = self.load("https://ulozto.net/reloadXapca.php",
                                   get={'rnd': timestamp()})
-
-                xapca = xapca.replace(
-                    'sound":"',
-                    'sound":"https:').replace(
-                    'image":"',
-                    'image":"https:')
                 self.log_debug("xapca: %s" % xapca)
 
                 data = json.loads(xapca)
-- 
2.30.2

